astropy/time/tests/test_basic.py

```diff
@@ -1352,3 +1352,14 @@ def test_setitem_deltas():
     t[1] = 3
     assert not hasattr(t, '_delta_tdb_tt')
     assert not hasattr(t, '_delta_ut1_utc')
+
+
+def test_subclass():
+    class _Time(Time):
```

Comment:
Could you add a docstring with a link back to the original issue and your PR? I.e., something like
```
"""Check that we can initialize subclasses with a Time instance."""
# Issue gh-7449 and PR gh-7453.
```
(For that extra commit, you can add "(ci skip)" to avoid rerunning travis, etc., and needlessly using up some kWhr.)